[BUG] Log entry (error) is writen to error.txt instead of sys.log

**Describe the bug:**
An error raised during compiling isn't written to the sys.log instead it's placed in the error.txt. The messages has a `>` in it. Resulting in this behavior 

A entry to the log is written by executing an [`echo` command ](https://github.com/Efratror/LS4P/blob/746171a300885c57ef5676d075a3e96db73858f8/server/src/scripts/syslogs.ts#L38)through a shell of some kind. The echo command is piped to the sys.log using `>>`. If the logEntry contains a `>` or `>>` then the pipe is pointing to the bit after it.

**Expected behavior:**
Log messages must never end up in the error.txt file.

**Possible solution:**
Find any `>` and escape it based on the type of OS currently in use.

**Additional context:**
Full error message:
```
2022-05-22T12:00:22.04 - [ERROR] Command failed: ...\LS4P/jre/bin/java --module compilerModule/com.compiler --cp  ...\Processing\Projects\Test\*; ...\LS4P\server\out/compile/ProcessingDefault.java > ...\Github\LS4P\server\out/compile/error.txt
Exception in thread "main" java.lang.IllegalArgumentException: Compilation unit is not of SOURCE kind: "--cp"
	at jdk.compiler/com.sun.tools.javac.api.JavacTool.getTask(Unknown Source)
	at jdk.compiler/com.sun.tools.javac.api.JavacTool.getTask(Unknown Source)
	at jdk.compiler/com.sun.tools.javac.api.JavacTool.getTask(Unknown Source)
	at compilerModule/com.compiler.main(Unknown Source)
```

--- a/server/src/scripts/syslogs.ts
+++ b/server/src/scripts/syslogs.ts
@@ -35,6 +35,13 @@ export async function write(message: string | unknown, logLevel : severity = sev
 		logEntry = `${datetime} - ${_message}`
 	}
 	
+	if(process.platform === 'win32') {
+		logEntry = logEntry.replace(/>/g, '^>')
+	}
+	else {
+		logEntry = logEntry.replace(/>/g, '\>')
+	}
+
 	exec(`echo ${logEntry} >> ${logPath}`)
 
 	if (logLevel == 3 || logLevel == 4 ) {

